Tes avi

fahrizalrahman opened the pull request:
Perbaikan Mater data produk

- (persediaan barang salah)
- Perbaikan datatable ajax

RindangRamadhan committed:
Aduan Implemntator

- membulatkkan subtotal_tbs pada file cek_subtotal_penjualan

RindangRamadhan committed:
Merge remote-tracking branch 'origin/tes_avi' into tes_avi

RindangRamadhan committed:
Aduan Implementator (Avi)

Memperbaiki perhitungan total akhir ketika ada potongan persen

rikopernando committed:
perbaikan persediaan barang

sama seperti di surya=> fitur edit (dblclick) belum berfungsi dengan baik masih bnayak yang salah dan cachenya belum otomatis terupdate

rikopernando committed:
perbaikan persediaan barang

sama seperti di surya=> fitur edit (dblclick) belum berfungsi dengan baik masih bnayak yang salah dan cachenya belum otomatis terupdate

rikopernando committed:
erbaikan pembayaran piutang => saat dihapus jumlah kredit di penjualan malah double. karena dua x di update(di code dan trigger), sudah dihapus yang dari kode.

erbaikan pembayaran piutang => saat dihapus jumlah kredit di penjualan
malah double. karena dua x di update(di code dan trigger), sudah dihapus
yang dari kode.

rikopernando committed:
Mengubah logika Laporan Penjualan rekap(Filter kategori barang)

Mengubah logika Laporan Penjualan rekap(Filter kategori barang)

rikopernando committed:
Merge remote-tracking branch 'origin/tes_avi' into tes_avi

fahrizalrahman committed:
Perbaikan Mater data produk 

- (persediaan barang salah)
- Perbaikan datatable ajax

fahrizalrahman committed:
Merge remote-tracking branch 'origin/tes_avi' into tes_avi

RindangRamadhan commented:
REVIEW CODE FAHRIZAL

update_barang.php
	1. Edit pada kolom di master data barang, tidak berfungsi.
barang.php
	1. SELECT * tolong dibenahi, ambil yang perlu saja

RindangRamadhan commented:
REVIEW CODE RIKO

update_barang.php
	1. Edit pada kolom di master data barang, tidak berfungsi.
persediaan_barang.php
	1. tombol hapus belum berfungsi
laporan penjualan rekap
	1. di keterangan showing (bagian bawah tabel) tertera "Showing 1 to 10 of 22 entries" tapi hanya 13 yang tampil
	2. ketika dicetak dan diekspor excel keterangan nilai totalnya masih salah, lalu penampilan kolom ditable ketika ekspor tidak sama dengan laporannya
	3.

fahrizalrahman committed:
Perbaikan review kode rama

- update barang klick dua kali tidak bisa update
- pembenahan ketika selesai konversi jasa link salah

fahrizalrahman committed:
kolom harga master barang produk

ganti menggunakan separator

fahrizalrahman committed:
Perbaikan Proses Edit Penjualan 

- delete detail penjualan sebelum insert ulang 
- tanggal mengikuti di tanggal form

rikopernando commented on proses_bayar_edit_jual.php:
```diff
@@ -23,19 +23,13 @@
             $sisa_kredit = angkadoang($_POST['jumlah_kredit_baru']);
```
-di file ini query yang double/atau yang sudah ada lebih baik dihapus saja. contoh :query untuk pelanggan, pajak, dll silahkan dilihat lagi mas.
- form edit penjualan nya, ketika tgl jatuh tempo nya kosong kemudian muncul alert, dan total akhir nya jadi kosong, hasunya jangan dikosong kan

fahrizalrahman committed:
perbaikan review kode (Riko)

- query double & pengambilan data double

fahrizalrahman committed:
Perbaikan Review Kode (2) RIco

tertinggal penghapusan double query

RindangRamadhan committed:
Perbaikan Bug (Avi)

- Memperbaiki insert detail_pemebayaran_piutang pada kolom total

RindangRamadhan committed:
Merge remote-tracking branch 'origin/tes_avi' into tes_avi

RindangRamadhan committed:
Cetak Penjualan Piutang

menambhkan Keterangan Diskon Persen pda Hasil Cetak Piutang
